controllers/servers/utils.py

```diff
@@ -577,6 +577,16 @@ def validate_addons_request(request, replication_type):
     if request.volume_id == "" or (replication_type == REPLICATION_TYPE_MIRROR and request.replication_id == ""):
         raise ValidationException(messages.VOLUME_ID_SHOULD_NOT_BE_EMPTY_MESSAGE)
 
+    if replication_type == REPLICATION_TYPE_EAR:
+        logger.debug("validating obsolete rcrelationship parameters")
+        if request.replication_id != "":
+            raise ValidationException(messages.INVALID_EAR_PARAMETER_MESSAGE.format(
+                servers_settings.PARAMETERS_REPLICATION_HANDLE))
+
+        if request.parameters.get(servers_settings.PARAMETERS_SYSTEM_ID):
+            raise ValidationException(messages.INVALID_EAR_PARAMETER_MESSAGE.format(
+                servers_settings.PARAMETERS_SYSTEM_ID))
+
```

Comment:
IMO you can export each `raise` "paragraph" to a function

Reply:
There are 4 raise "paragraphs" inside validate_addons_request function. Do you want to separate this function to 4 different functions?

Reply:
yes it will be great if you can do it in this PR, but not all the "paragraphs" was changed in this PR so if you think it needs a separate ticket for it you can do it

Reply:
I added two separate functions